html/semantics/forms/form-control-infrastructure/form.html

```diff
@@ -10,37 +10,79 @@
 <p><fieldset id="fieldset">fieldset</fieldset>
 <p><input id="input">
 <p><keygen id="keygen">
-<p><label id="label">label</label>
 <p><object id="object">object</object>
 <p><output id="output">output</output>
 <p><select id="select"><option>select</option></select>
 <p><textarea id="textarea">textarea</textarea>
+
+<!-- label is special: label.form is an alias for label.control.form -->
+<p><label id="label">label</label>
+<p><label id="label-form" form="form">label-form</label>
+<p><label id="label-form-form2" form="form2">label-form-form2</label>
+<p><label id="label-with-control">label-with-control <input></label>
+<p><label id="label-for" for="control-for-label">label-for</label> <input id="control-for-label">
 </form>
```

Comment:
We should add one more here with a label pointing to an input that has a form attribute which points to form2. It's sort of tested by "form2" below, but there the control is not nested inside another form.